### PR TITLE
Create vpc_service_controls_gcp.adoc

### PR DESCRIPTION
**Pull request by felix-melligan**
Added information regarding VPC-Service Controls based on my investigation. Includes information about ONTAP images and rules required to work with VPC-SC.

### CODE REVIEW

File: reference_vpc_service_controls_gcp.adoc [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+* The user who deploys Cloud Manager will need to be able to reference an image hosted in the projectId "netapp-cloudmanager", project number "14190056516".
++
+To deploy Cloud Volumes ONTAP:
++
+* The Cloud Manager service account will need to be able to reference an image hosted in the projectId "netapp-cloudmanager", project number "14190056516" from the service project.
+*The default Google APIs Service Agent service account will need to be able to reference an image hosted in the projectId "netapp-cloudmanager", project number "14190056516" from the service project.
++
+Examples of the rules needed for pulling these images with VPC-SC are defined below.
+
+== VPC-SC and Private Service Connect
```
Comment (netapp-bcammett) [inline reference_vpc_service_controls_gcp.adoc:49]: @felix-melligan, the previous sections were just about VPC Service Controls. What's the relationship between VPC Service Controls and Private Service Connect? Are they always used together?

File: reference_vpc_service_controls_gcp.adoc [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+As referenced in other parts of the documentation, Google Private Service Connect is not supported with Cloud Tiering or Cloud Backup service currently.
+
+== Policies
```
Comment (netapp-bcammett) [inline reference_vpc_service_controls_gcp.adoc:53]: @felix-melligan, what kind of policy is this?

File: reference_vpc_service_controls_gcp.adoc [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+=== Ingress
+
+From:
```
Comment (netapp-bcammett) [inline reference_vpc_service_controls_gcp.adoc:57]: @felix-melligan, How are these policies created in Google Cloud?

For example, does a user provide the following text in the syntax shown below?

File: reference_vpc_service_controls_gcp.adoc (hunk 2 above)
Comment (felix-melligan) [reply reference_vpc_service_controls_gcp.adoc:53]: Added description

File: reference_vpc_service_controls_gcp.adoc (hunk 3 above)
Comment (felix-melligan) [reply reference_vpc_service_controls_gcp.adoc:57]: Fixing this through extra description as well as formatting (This is how it appears in the config of VPC-SC in the GCP console)

File: reference_vpc_service_controls_gcp.adoc (hunk 1 above)
Comment (felix-melligan) [reply reference_vpc_service_controls_gcp.adoc:49]: Not always, but they're often used together so thought I'd mention it in the same section. Could be removed

**Pull request merged**